ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -286,6 +292,35 @@ spec:
         - name: ocp-registry-kubeconfig
           workspace: ocp-registry-kubeconfig
 
+    # Trigger IBM webhook for marketplace replication
+    - name: trigger-marketplace-replication
+      runAfter:
+        - publish-to-ocp-registry
```

Comment:
I think we also require the task `trigger-marketplace-replication` should run after the `operator-validation` task.

Reply:
Yeah it should, since it depends on request from `operator-validation`. Good catch! Added now.

Reply:
Seems like it. While this would work it's nice to see the explicit dependencies in the diagram.